codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocol.kt

```diff
@@ -357,5 +357,5 @@ class ServerRpcV2Protocol(
 
     override fun serverRouterRuntimeConstructor() = "rpc_v2_router"
 
-    override fun serverContentTypeCheckNoModeledInput() = true
+    override fun serverContentTypeCheckNoModeledInput() = false
```

Comment:
We need to update the comment in the `ServerProtocol` `interface` which calls out `rpcv2` and `restJson1`.